test cases/objc/2 nsstring/meson.build

```diff
@@ -2,6 +2,8 @@ project('nsstring', 'objc')
 
 if host_machine.system() == 'darwin'
   dep = dependency('appleframeworks', modules : 'foundation')
+elif host_machine.system() == 'cygwin'
+  error('MESON_SKIP_TEST GNUstep isn't packaged for Cygwin.')
```

Comment:
There's actually a typo here; you're terminating the `'` string with `isn't`. Not sure why it isn't raising an exception...